src/main/js/components/subcomponents/MenuItem.js

```diff
@@ -320,6 +322,41 @@ class ManagerUpdateMenuItemDialog extends React.Component {
     }
 
 
+
+    //updateCustomerCart(menuItem){
```

Comment:
Functionality was moved to `App.js` is it ok for me to remove this block from here? @EBruchet